tests/lvmo/test_lvm_alerts.py

```diff
@@ -0,0 +1,228 @@
+    def test_thin_pool_capacity_alert(
+        self, namespace, init_lvm, storageclass, pvc, pod
+    ):
+        """
+
+        Test to verify thin pool capacity alert:
+        1. run io up to 70, check alerts - no alert expected
+        2. run io up to 76, check alerts
+        3. run io up to 86, check alerts - critical alert expected
+
+        """
+        log.info("Test Started successfully")
+        log.info(f"LVMCluster version is {self.lvm.get_lvm_version()}")
+        log.info(
+            f"Lvm thin-pool overprovisionRation is {self.lvm.get_lvm_thin_pool_config_overprovision_ratio()}"
+        )
+        log.info(
+            f"Lvm thin-pool sizePrecent is {self.lvm.get_lvm_thin_pool_config_size_percent()}"
+        )
+        size_to_70 = f"{int(float(self.thin_pool_size)*0.7)}Gi"
+        size_to_76 = f"{int(float(self.thin_pool_size)*0.06)}Gi"
+        size_to_86 = f"{int(float(self.thin_pool_size)*0.1)}Gi"
+        sizes_list = [
+            {
+                "size_to_fill": size_to_70,
+                "file_name": "run-to-70",
+                "pvc_expected_size": f"{float(self.pvc_size)*0.7}",
+                "alert": None,
+            },
+            {
+                "size_to_fill": size_to_76,
+                "file_name": "run-to-76",
+                "pvc_expected_size": f"{float(self.pvc_size)*0.76}",
+                "alert": topolvm_alerts.get("tp_data_75_precent"),
+            },
+            {
+                "size_to_fill": size_to_86,
+                "file_name": "run-to-86",
+                "pvc_expected_size": f"{float(self.pvc_size)*0.86}",
+                "alert": topolvm_alerts.get("tp_data_85_precent"),
+            },
+        ]
+
+        log.info(f"LV Size:{self.thin_pool_size}")
+        self.metric_data = dict()
+        for size in sizes_list:
+            log.info(
+                f"{size.get('size_to_fill')}, {size.get('file_name')}, {size.get('pvc_expected_size')}"
+            )
+            self.pod_obj.run_io(
+                storage_type="fs",
+                size=size.get("size_to_fill"),
+                rw_ratio=0,
+                jobs=1,
+                runtime=0,
+                depth=4,
+                rate="1250m",
+                rate_process=None,
+                fio_filename=size.get("file_name"),
+                bs="100M",
+                end_fsync=0,
+                invalidate=None,
+                buffer_pattern=None,
+                readwrite="write",
+                direct=1,
+                verify=False,
+            )
+            self.pod_obj.get_fio_results()
+            # This is a workaround for BZ-2108018
+            log.info("Wait 10 minutes for metrics refresh interval")
+            time.sleep(600)
+            # End of workaround
+            self.lvm.compare_percent_data_from_pvc(
+                self.pvc_obj, float(size["pvc_expected_size"])
+            )
+
+            val = parse_topolvm_metrics(topolvm_metrics)
```

Comment:
Instead of waiting 10 minutes we can do some Timeoutsampler to pull the metric 

```
from datetime import datetime
now = datetime.now()
start_time = now.strftime("%H:%M:%S")
time_list = []
metric_list = []
first_results = parse_topolvm_metrics(topolvm_metrics)
metric_list.append(first_results)
time_list.append(start_time)
for metric_reults in TimeoutSampler(
                                    timeout=700,
                                    sleep=2,
                                    func=parse_topolvm_metrics,
                                    topolvm_metrics,
                                    ):

<compare the metrics from the metric_reults to first_results>
if some change in metric:
    metric_list.append(metric_reults)
    time_list.append(now.strftime("%H:%M:%S"))
    break
    
```